packages/cli/src/util/get-pagination-opts.ts

```diff
@@ -0,0 +1,21 @@
+export interface PaginationOptions {
+  '--next'?: number;
+  '--limit'?: number;
+}
+
+export function getPaginationOpts(opts: PaginationOptions) {
+  const { '--next': nextTimestamp, '--limit': limit } = opts;
+
+  if (typeof nextTimestamp !== undefined && Number.isNaN(nextTimestamp)) {
+    throw new Error('Please provide a number for flag --next');
```

Comment:
```suggestion
    throw new Error('Please provide a number for option --next');
```

We call this options (not flags) 

See https://vercel.com/docs/cli/alias#unique-options